src/Hedgehog/Property.fs

```diff
@@ -151,7 +151,12 @@ module private Pretty =
 
         List.iter (append sb) (Journal.toList journal)
 
+#if FABLE_COMPILER
+        let str = sb.ToString()
+        str.Substring(0, str.Length-1)
+#else
         sb.ToString(0, sb.Length - 1) // exclude extra newline
+#endif        
```

Comment:
Here, Fable also didn't implement ToString with offset and length. The code compiled by fable will probably be not as performant as the .NET one. Do you have any idea how can we solve this?

Reply:
Perhaps we can change the original one by adding a comment also

```
(sb.ToString()).Substring(0, str.Length-1)
```

Reply:
Closing this conversation. Fable has ToString with offset and index now.